### PR TITLE
Update Face README.python.md

### PR DESCRIPTION
**Pull request by tjprescott**
Updated so that you can actually generate the SDK simply with
`autorest readme.md --python`

### CODE REVIEW

**Comment by tjprescott** [comment]
@jhendrixMSFT removing the labels added by the bot as false positives. This doesn't affect Go at all, and it does not break the Python SDK (but using this readme in its current state certainly would).

File: specification/cognitiveservices/data-plane/Face/readme.python.md [hunk 1]
```diff
@@ -6,21 +6,23 @@ Use `--python-mode=update` if you already have a setup.py and just want to updat
...
+license-header: MICROSOFT_MIT_NO_VERSION
+add-credentials: true
+payload-flattening-threshold: 2
+namespace: azure.cognitiveservices.vision.face
+package-name: azure-cognitiveservices-vision-face
+clear-output-folder: false
+use: "@microsoft.azure/autorest.python@~4.0.71" 
+version: V2
+multiapi: true
+no-async: true
```
Comment (tjprescott) [inline specification/cognitiveservices/data-plane/Face/readme.python.md:18]: Autorest will generate the completely wrong set of files without these settings.

File: specification/cognitiveservices/data-plane/Face/readme.python.md [hunk 2]
```diff
@@ -6,21 +6,23 @@ Use `--python-mode=update` if you already have a setup.py and just want to updat
...
 ```
 ``` yaml $(python) && $(python-mode) == 'update'
-python:
-  no-namespace-folders: true
-  output-folder: $(python-sdks-folder)/cognitiveservices/azure-cognitiveservices-vision-face/azure/cognitiveservices/vision/face
+no-namespace-folders: true
+output-folder: $(python-sdks-folder)/cognitiveservices/azure-cognitiveservices-vision-face/azure/cognitiveservices/vision/face
+keepVersionFile: true
```
Comment (iscai-msft) [inline specification/cognitiveservices/data-plane/Face/readme.python.md:23]: should be `keep-version-file`

Comment (tjprescott) [reply specification/cognitiveservices/data-plane/Face/readme.python.md]: Fixed.

**Review by iscai-msft — APPROVED** [review]
looks good. for documentations sake: this is to only generate a track 1 sdk, hence the differences in `use`, and `version`

**Pull request merged**